etc/profile-a-l/anki.profile

```diff
@@ -17,6 +17,9 @@ noblacklist ${HOME}/.config/mpv
 noblacklist ${HOME}/.local/share/Anki2
 noblacklist ${HOME}/.mplayer
 
+# sh and dbus-send are used by aqt/theme.py to query dark mode through org.freedesktop.portal.Desktop
+# Allow /bin/sh (blacklisted by disable-shell.inc)
+include allow-bin-sh.inc
 # Allow lua (blacklisted by disable-interpreters.inc)
 include allow-lua.inc
```

Comment:
```suggestion
# sh and dbus-send are used by aqt/theme.py to query dark mode through
# org.freedesktop.portal.Desktop.
# Allow /bin/sh (blacklisted by disable-shell.inc)
include allow-bin-sh.inc

# Allow lua (blacklisted by disable-interpreters.inc)
include allow-lua.inc
```

Reply:
By the way, part of a recent suggestion was apparently missed when it was
applied, so please use the "Commit suggestion" button above to add the entire
suggestion as is to the branch.  Then you can just git pull and squash/edit
locally.

This also applies to the other PRs.